packages/design-system/src/components/text.ts

```diff
@@ -6,6 +6,7 @@ export { textVariants };
 export const textStyle = css({
   margin: 0, // in case it's used with <p>
   WebkitFontSmoothing: "antialiased",
+  userSelect: "none",
   variants: {
     variant: textVariants,
```

Comment:
> Or maybe in some global styles like *:not(input, textarea) {user-select:none}.

would you prefer this instead of locally defining? I feel always dirty doing things like this, ideally we would have a normalize file and would spread the definitions into each component, similar to how we do it in sdk

Reply:
I added it to a few places, didn't seem like a huge deal to maintain, do you think it's inconsistent and going to be hard to maintain?

Reply:
Yeah, I think it's ok as is too. No strong opinions

Reply:
On one hand it's kinda global, but in reality, it's only needed where we render text and that always goes over those textVariants  and text component, so kinda not needed to be global, right?

I am thinking maybe indeed adding it to textVariants before reexporting without changing the original object is a better tradeoff?

Reply:
`textVariants` seem a bit better to me that the current solution, yes. Because it automatically applies it in more places.

But need to be careful to make sure we override it back to `auto` in `input/textarea`